sfcasts/dashboard.md

```diff
@@ -1,95 +1,61 @@
-some links or some graphs or something, by the way, styling is all done in bootstrap 5
-with FontAwesome.
-
-So let's see if we can customize the dashboard a little bit. Probably the best thing
-about easy admin is that all the config is done in PHP, usually via methods in your
-controller. So for example, wanna configure the dashboard. There is a configured
-dashboard method for that. So we can change this, the title of the page two call
-during overflow admin. And I wanna refresh, we get Coran overflow admin, and there
-are a number of other methods. You can just look at the, um, uh, auto complete here.
-So, uh, related to the fab icon path, there's various ways that you can modify
-things. You can see there's things about the sidebar being minimized. That's
-referring to a nice little feature here where you can double click on this little
-thing to expand or, uh, collapse or expand the sidebar.
-
-But really the main part of the dashboard are these menu items over here, which we
-only have one right now, and this is controlled by configure menu items. So just to
-show that we can let's change this FA to `fa-dashboard.` Now these are font awesome.
-This leverages the font awesome library. So `fa-dashboard] corresponds to that little
-icon right there. All right. So we can definitely do more with our dashboard, but
-that's enough for now because what we're really here for are the CRUD controllers.
-These are these sections of our site that it's going to allow us to create, read,
-update, and delete all of our entities. Let's get those going next.
+As a reminder, `symfony console` is exactly the same as running `php bin/console`. The only difference is that running `symfony console` allows the Docker environment variables to be injected into this command. It generally makes no difference unless you're running a command that requires database access. So in this case, `php bin/console` would work just fine.
+
+I'll stick with `symfony console` throughout this tutorial. So say:
+
+```terminal
+symfony console make:admin:dashboard
+```
+
+We'll call it "DashboardController."
+
+Sounds good! To generate it, say `src/Controller/admin` and... done! This creates one new file for us: `src/Controller/Admin/DashboardController.php`.
+
+Let's go check it out!
+
+Ooh... Let me see... There's not a lot here, but one thing you might notice is that it has a route for `/admin`. So now if I go over here to `/admin`... we do hit the admin dashboard!
+
+Okay, so there's a few things to notice here. First is that we do have a `/admin` route. This isn't fancy, and there's really nothing "EasyAdmin" about it. This is just how we create routes in Symfony. This is a PHP 8 attribute route, which you may or may not be familiar with. I've typically used annotations until now, but now that I'm using PHP 8, I'll be using attributes instead of annotations. Don't worry, though! They work exactly the same. If you're still using PHP 7, then you can use annotations just fine.
```

Comment:
`First is that` -> `The first is that`

I'm lazy when I record. This looks and feels better.

> annotations until now, but now that...

I have 2x "now" in 3 words, so we should change something to avoid that :)